ARTICLE-TEMPLATE.mdx

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
Add a comment on the top, asking users to copy the raw version of the file before applying them

Reply:
sorry did not get you can you elaborate

Reply:
Basically we want the users to open the raw, un-rendered version of the file for their purpose. Not the rendered file. So having a comment at the top would be helpful